src/api/app/models/staging/staged_requests.rb

```diff
@@ -11,10 +11,18 @@ def create
         link_package(bs_request_action)
       end
       # TODO: implement delete requests
```

Comment:
Is there anything left to do to implement delete requests? Does this comment still apply?

Reply:
Well, at least it's to be decided - because the client still does this:
https://github.com/openSUSE/openSUSE-release-tools/blob/0d0290f1e6a4fc53f76d377a8b5686a714751e69/osclib/stagingapi.py#L1098

Reply:
(and there are quite some [corner cases](https://github.com/openSUSE/openSUSE-release-tools/issues/1899) where this fails, so I would actually leave it in the client)